signin/index.html

```diff
@@ -8,45 +8,44 @@
   <title>로그인</title>
 </head>
 <body>
-  <div class="login">
-    <header>
-      <div>
-        <a href="/"><img class="header-logo" src="/assets/image/logo.png"></a>
-      </div>
-      <div class="header-signup">
-        회원이 아니신가요? <a class="signup" href="/signup">회원 가입하기</a>
-      </div>
-    </header>
-    <section>
-      <div class="contents-frame">
-        <form class="contents-form" action="/folder.html" onsubmit="return login()">
-          <label for="email">이메일</label>
-          <input type="email" name="email" id="email" placeholder="이메일">
-          <label for="password">비밀번호</label>
-          <div class="password-container">
-            <div>
-              <input type="password" name="password" id="password" placeholder="비밀번호">
-            </div>
-            <div class="password-eye-container">
-              <img class="password-eye" src="/assets/image/eye-on.png">
-            </div>
+  <header>
+    <div>
+      <a href="/"><img class="header-logo" src="/assets/image/logo.png"></a>
+    </div>
+    <div class="header-signup">
```

Comment:
저희 클래스명 규칙을 카멜 케이스로 맞추기로 해서 이 부분 반영해주심 좋을 거 같아요!
```suggestion
    <div class="headerSignup">
```